### PR TITLE
Development -> Master

### PR DESCRIPTION
**Pull request by matsinet**

### CODE REVIEW

File: Section07/7.2-APIsAndAxios1.md [hunk 1]
```diff
@@ -242,11 +242,57 @@ axios
...
+---
+
+<br>
+
 ## API Authorization
 
 There are many open APIs, but most of them are not _quite_ as open as the placeholder JSON data API that we worked with earlier.
 
-While computing power is cheap, it's not _free_, and most APIs require at least a little bit of hoop-jumping to get to their data.
+While computing power is cheap, it's not _free_, and most APIs require at least a little of hoop-jumping to get to their data.
```
Comment (H-Matzy) [inline Section07/7.2-APIsAndAxios1.md:295]: "at least a little of hoop jumping" might need to be "at least a little hoop jumping" omitting the word "of" now that we have deleted "bit"

**Review by H-Matzy — APPROVED** [review]
Looks good to me

**Comment by matsinet** [comment]
> All looks good except aren't we using state as a parameter here instead of st?

Yep, it has been updated

**Review by JohanBester — APPROVED** [review]
Approved - merge it  :)

**Pull request merged**